Teams enhancements (#24947)
demisto/content: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

Use the Microsoft Teams integration to send messages and notifications to your team members and create meetings.
This integration was integrated and tested with version 1.0 of Microsoft Teams.

**Note::** 

> - The integration has the ability to run builtin XSOAR commands, through a mirrored channel. make sure to pass the command in the chat exactly as typed in CORTEX XSOAR CLI. example: `!DeleteContext all=yes`. Use command `mirror-investigation` to mirror/create a mirrored channel.

Reviewer: ```suggestion
- The integration has the ability to run built-in Cortex XSOAR commands, through a mirrored channel. Make sure to pass the command in the chat exactly as typed in the CORTEX XSOAR CLI. For example: `!DeleteContext all=yes`. Use the command `mirror-investigation` to mirror/create a mirrored channel.
```

> - For use cases where it is only need to send messages to a specific channel, we recommend checking the [Microsoft Teams via Webhook Integration](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook), which has a simpler setup.

Reviewer: ```suggestion
- For use cases where it is only needed to send messages to a specific channel, we recommend checking the [Microsoft Teams via Webhook Integration](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook), which has a simpler setup.
```

- Posting a message or adaptive card to a private/shared channel is currently not supported in the ***send-notification*** command. Thus, also the ***mirror_investigation*** command does not support private/shared channels. For more information, see [Microsoft General known issues and limitations](https://learn.microsoft.com/en-us/connectors/teams/#general-known-issues-and-limitations).
- In case of multiple chats/users sharing the same name, the first one will be taken.
- See Microsoft documentation for [Limits and specifications for Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/limits-specifications-teams).

> - If a non XSOAR user ran the command `new incident` in the chat with the bot, the owner of the created incident would be the logged in XSOAR user, not the external user who ran the command.

Reviewer: ```suggestion
- If a non-Cortex XSOAR user ran the `new incident` command in the chat with the bot, the owner of the created incident would be the logged in Cortex XSOAR user, not the external user who ran the command.
```

***
Mirrors the Cortex XSOAR investigation to the specified Microsoft Teams channel. Supports only standard channels.

> **Note**: Mirrored channels could be used to run CORTEX XSOAR builtin commands.

Reviewer: ```suggestion
**Note**: Mirrored channels could be used to run Cortex XSOAR built-in commands.
```

    
    Try inserting your configured message endpoint in a browser tap, click `Enter`, if `Method Not Allowed` is returned, the endpoint is valid and ready to communicate, otherwise, it needs to be handled according to the returned error's message.

>     In some cases, a connection is not created between Teams and the messaging endpoint, when adding a bot to the team. You can work around this problem by adding any member to the team the bot was added to (the bot should be already added to the team). It will trigger a connection and solve the issue, then you can remove the member that was added with no poroblems.

Reviewer: ```suggestion
    In some cases, a connection is not created between Teams and the messaging endpoint, when adding a bot to the team. You can work around this problem by adding any member to the team the bot was added to (the bot should be already added to the team). This will trigger a connection and solve the issue. You can then remove the member that was added.
```
